README.md

```diff
@@ -245,6 +252,9 @@ if os() ~~ /centos/ {
 }
 ```
 
+### 5. Non-Blocking Asynchronous Operations
+All Sparrow6 evaluations run in background threads, ensuring systemd's core operations continue at full speed.
```

Comment:
Worth it mention that this background thread is executed by some external (monitoring) tool?